cmd/wallet-web/src/translations/CredentialDetails/fr.json

```diff
@@ -1,5 +1,5 @@
 {
-  "allVaultLabel": "Ajouter une chambre forte",
```

Comment:
I changed the AllVaultLabel french translation from `Ajouter une chambre forte` -> `Toutes les chambres fortes` based on  my (limited) French and [this abstract](https://app.abstract.com/projects/9ec92568-60a2-4fee-8215-5aa997d52428/branches/dc10bcee-d9a1-44f9-b644-8f843c6f1a19/commits/68d43e7308faabf1b365be94685d85413c7ad001/files/95d5757f-f10e-43f6-ad50-0faa236217fd/layers/B18B0F73-D1C3-4BE9-849F-6EADB710BA59?collectionId=ba692773-75df-473a-a41c-38a0647809a4&collectionLayerId=595d57fd-3a25-405f-9db8-113cddcd1d67) page, please correct me if I am wrong!

(screenshot of Abstract Credentials Page: )
![image](https://user-images.githubusercontent.com/44453261/173067755-18ec89d9-347a-4396-b0e3-a01b98540dd7.png)

Reply:
Can we confirm this with ux team ?